src/CoreWCF.Primitives/src/CoreWCF/Security/SecurityMessageProperty.cs

```diff
@@ -191,7 +200,7 @@ public IMessageProperty CreateCopy()
         public static SecurityMessageProperty GetOrCreate(Message message)
         {
             if (message == null)
-                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull(nameof(message));
+                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("message");
```

Comment:
You reverted a nameof

Reply:
:) sorry.. as I have said mostly find/replace